components/Footer.jsx

```diff
@@ -0,0 +1,154 @@
+                    key="joinmastodon.org"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href="https://docs.joinmastodon.org"
+                  >
+                    <FormattedMessage
+                      id="nav.docs"
+                      defaultMessage="Documentation"
+                    />
+                  </a>,
+                  <a
+                    className="py-2 inline-block"
+                    key="mastodon/discussions"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href="https://github.com/mastodon/mastodon/discussions"
+                  >
+                    <FormattedMessage
+                      id="nav.support"
+                      defaultMessage="Support"
+                    />
+                  </a>,
+                ],
+              },
+              {
+                heading: (
+                  <FormattedMessage id="nav.company" defaultMessage="Company" />
+                ),
+                links: [
+                  <Link key="/imprint" href="/imprint">
+                    <a className="py-2 inline-block">
+                      <FormattedMessage
+                        id="contact_us"
+                        defaultMessage="Contact us"
+                      />
+                    </a>
+                  </Link>,
+                  <a
+                    className="py-2 inline-block"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    key="https://blog.joinmastodon.org"
+                    href="https://blog.joinmastodon.org/"
+                  >
+                    <FormattedMessage id="nav.blog" defaultMessage="Blog" />
+                  </a>,
+                  <a
+                    className="py-2 inline-block"
+                    key="/press-kit.zip"
+                    href="/press-kit.zip"
+                  >
+                    <FormattedMessage
+                      id="credits.press_kit"
+                      defaultMessage="Press kit"
+                    />
+                  </a>,
+                  <Link key="/privacy-policy" href="/privacy-policy">
+                    <a className="py-2 inline-block">
+                      <FormattedMessage
+                        id="privacy_policy"
+                        defaultMessage="Privacy policy"
+                      />
+                    </a>
+                  </Link>,
+                ],
+              },
+            ].map((menu) => (
+              <div className="flex flex-col gap-2" key={menu.heading}>
+                <h4 className="py-1 h6">{menu.heading}</h4>
+                <ul className="b2 flex flex-col m-0 p-0 gap-2">
+                  {menu.links.map((link) => (
+                    <li key={link.key}>{link}</li>
+                  ))}
+                </ul>
+              </div>
+            ))}
```

Comment:
did this loosey goosey loop so i don't have to write AS many classes. works as long as the content is relatively consistent here

Reply:
looks good! I was thinking if there were other ways to write this loop, but given the use of the `Link` + `FormattedMessage` here, this is a good structure.